pylxd/image.py

```diff
@@ -80,7 +81,66 @@ def create(cls, client, image_data, public=False, wait=False):
             Operation.wait_for_operation(client, response.json()['operation'])
         return cls(client, fingerprint=fingerprint)
 
+    @classmethod
+    def create_from_config(cls, client, config, wait=False):
```

Comment:
I don't really care for the "pass in a random config as a parameter", particularly because later on, there are requirements enforced on the config itself. I think I'd rather have a `copy_from_host` and 'create_from_simplestream` class methods. For example, `copy_from_host` would take a `name` and another `client` as parameters, and already that makes for a cleaner interface.